### PR TITLE
pythonpath solution

### PR DESCRIPTION
**Pull request by MarcelWilnicki**

### CODE REVIEW

**Comment by MarcelWilnicki** [comment]
@jangrzybek take a look now.

File: computer_vision/classification/alexnet/run.py [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
 import argparse
 
 import torch
 import torchvision
+from pathlib import Path
+
+try:
+    from utils.cv.imagenet import ImageNet
+except ModuleNotFoundError as e:
+    pythonpath = os.path.join(Path(os.getcwd(), __file__))
```
Comment (jangrzybek) [inline computer_vision/classification/alexnet/run.py:15]: __file__ gives you what you need already. No need for os.getcwd(). Also, why not just use an already working solution from mzdev?

Comment (jangrzybek) [reply computer_vision/classification/alexnet/run.py:15]: https://github.com/AmpereComputingAI/model_zoo_dev/blob/main/downloads/utils.py

Comment (jangrzybek) [reply computer_vision/classification/alexnet/run.py:15]: Lets create a util under utils setting the python_path with an unified approach that could be imported by every runner.

Comment (MarcelWilnicki) [reply computer_vision/classification/alexnet/run.py:15]: __file__ returns the path to the file which is run from your working directory. If the user enters folder "alexnet", and runs the script from there "__file__" will return "run.py", the way I did this it always returns an abosolute path to the file which is run. From this path I can then set the pythonpath in the run script.

If the pythonpath is not set then such a function in utils will not be importable and thus will not work, or am I missing something?

Comment (jangrzybek) [reply computer_vision/classification/alexnet/run.py:15]: You are of course right on inability to use utils without setting PYTHON_PATH first.

But __file__ seems to return absolute path of file no matter where you run it:

> jan@g-workstation:~/Downloads$ cat test.py 
> print(__file__)
> jan@g-workstation:~/Downloads$ python3 test.py 
> **/home/jan/Downloads/test.py**
> jan@g-workstation:~/Downloads$ cd ../..
> jan@g-workstation:/home$ python3 jan/Downloads/test.py
> **/home/jan/Downloads/test.py**

Comment (jangrzybek) [reply computer_vision/classification/alexnet/run.py:15]: So, os.getcwd() is redundant.

Comment (MarcelWilnicki) [reply computer_vision/classification/alexnet/run.py]: strange, I get a different behavior, nonetheless os.getcwd() was redundant, I solved it in a different way

**Review by jangrzybek — APPROVED** [review]

**Review by jan-grzybek-ampere — APPROVED** [review]
LGTM

**Comment by jan-grzybek-ampere** [comment]
What's the status here? I think it should be merged some time ago as a good improvement.

**Comment by MarcelWilnicki** [comment]
> What's the status here? I think it should be merged some time ago as a good improvement.

I think it is good to go, but I will have to test it first.

**Comment by jan-grzybek-ampere** [comment]
@MarcelWilnicki do we want to merge this? is this tested? definitely needs rebasing now

**Comment by MarcelWilnicki** [comment]
I will look into this when I have some progress with the Bert model, maybe still this week, or the next

**Comment by MarcelWilnicki** [comment]
using setup scripts and sourcing the environment paths I think this update is no longer necessary.

**Pull request closed without merging**